ts/client/src/accounts/healthCache.ts

```diff
@@ -270,6 +297,18 @@ export class HealthCache {
     return this.healthSum(healthType, tokenBalances);
   }
 
+  public healthExcludingToken(
+    healthType: HealthType,
+    skipTokenIndex: TokenIndex,
+  ): I80F48 {
+    const tokenBalances = this.effectiveTokenBalancesInternal(
+      healthType,
+      false,
+      skipTokenIndex,
+    );
+    return this.healthSum(healthType, tokenBalances);
```

Comment:
hmm do you mean that this skipToken thing is not enough? Because even if skip this token, it was probably involved in spot markets, which we still considered

![image](https://github.com/blockworks-foundation/mango-v4/assets/89031858/398e88cb-9835-4bce-939b-cc84301dd074)